src/pages/SignUp/SignUp.scss

```
@@ -0,0 +1,142 @@
+@mixin errorMessage {
+  margin-left: 22px;
+  font-size: 13px;
+}
+@mixin signUpEndButton {
+  border: 1px solid #ccc;
+  min-width: 150px;
+  max-width: 150px;
+  height: 40px;
+  font-size: 17px;
+}
+
+@mixin tableDesign {
+  border-collapse: collapse;
+  border-spacing: 0;
+  border-top: 1px solid black;
+}
+
+@mixin thDesign {
+  color: #333;
+  background: #f3f3f3;
+  padding: 10px 0 10px;
+  border-bottom: 1px solid #ddd;
+  text-align: left;
+  width: 300px;
+}
+
+.signUp {
+  position: relative;
+  min-width: 1000px;
+  margin-bottom: 50px;
+  .signUpHeader {
+    margin: 50px;
+    .signUpTitle {
+      text-align: left;
+      font-family: 'KimjungchulGothic-Bold';
+      font-size: 35px;
+      border-bottom: 1px solid black;
+      margin-left: 225px;
+      margin-right: 190px;
+      padding-top: 55px;
+      color: black;
+    }
+  }
+  .infoFirst {
```

스타일 파일 전체적으로 다음 사항 통일시켜주세요!
선택자 사이에도 줄간격을 일정하게 넣어주세여야 합니다!
```suggestion
.signUp {
  position: relative;
  min-width: 1000px;
  margin-bottom: 50px;
  
  .signUpHeader {
    margin: 50px;
    
    .signUpTitle {
      text-align: left;
      font-family: 'KimjungchulGothic-Bold';
      font-size: 35px;
      border-bottom: 1px solid black;
      margin-left: 225px;
      margin-right: 190px;
      padding-top: 55px;
      color: black;
    }
  }
  
  .infoFirst {
```

---

모두 수정하였습니다!